client/state/notices/middleware.js

```diff
@@ -141,6 +143,18 @@ export const onPublicizeConnectionUpdateFailure = ( dispatch, { error } ) => dis
 	} ) )
 );
 
+export function onSiteSettingsSaveFailed( dispatch, action ) {
+	let text;
+	switch ( action.error.error ) {
+		case 'invalid_ip':
+			text = translate( 'One of your IP Addresses was invalid. Please try again.' );
```

Comment:
Yeah, This errors looked confusing to me as well, but I kept it to be backwards compatible. I thought there maybe a valid reason to keep it